### PR TITLE
Added total accounts balance. Improved getting account balance.

### PR DESCRIPTION
**Pull request by ndemidovich-sc**

### CODE REVIEW

File: electron/ironfish/IronFishManager.ts [hunk 1]
```diff
@@ -38,22 +38,36 @@ class AccountManager implements IIronfishAccountManager {
...
+    const accounts = this.node.wallet
+      .listAccounts()
+      .filter(
+        account =>
+          !search ||
+          account.name.includes(search) ||
+          account.publicAddress.includes(search)
+      )
+
+    for (const account of accounts) {
```
Comment (azhurauski-sc) [inline electron/ironfish/IronFishManager.ts:52]: Can we use Promise.all here?

File: electron/ironfish/IronFishManager.ts [hunk 2]
```diff
@@ -38,22 +38,36 @@ class AccountManager implements IIronfishAccountManager {
...
+        (a, b) =>
+          (SortType.ASC === sort ? 1 : -1) *
+          (Number(a.balance.confirmed) - Number(b.balance.confirmed))
+      )
+    }
+
+    return Promise.resolve(result)
   }
 
   get(id: string): Promise<AccountValue | null> {
```
Comment (azhurauski-sc) [inline electron/ironfish/IronFishManager.ts:73]: Should we also update this function, so the Account in our app will have same structure across the app?

File: electron/ironfish/IronFishManager.ts (hunk 1 above)
Comment (ndemidovich-sc) [reply electron/ironfish/IronFishManager.ts]: Done.

File: electron/ironfish/IronFishManager.ts (hunk 2 above)
Comment (ndemidovich-sc) [reply electron/ironfish/IronFishManager.ts]: Done.

**Review by azhurauski-sc — APPROVED** [review]

**Review by azhurauski-sc — APPROVED** [review]

**Pull request merged**